tableauserverclient/server/request_options_base.py

```diff
@@ -0,0 +1,9 @@
+import abc
+
+
+class RequestOptionsBase(object):
```

Comment:
Thank you!

Yeah, in this case it's not warranted (It's usually only done for large classes like a protocol in asyncio or twisted or something)

SIDE NOTE: I'm not a fan of the idea of having subclasses for all of our different url options/parameters, I feel like it'll just grow into a long list of "Which class do I need again", but in absence of a better idea I'll be quiet for now :)